### PR TITLE
Zero state ct sprint 1.13

### PR DESCRIPTION
**Pull request by CristalWilsonLobo**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: docker.local/config/conductor.zero_state_authorizer_recovery.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          key: total_rewards
+          monotonicity: increase
+          timeout: "2m"          
+      - stop: ['authorizer-1']
+      - command:
+          name: "cleanup_provider"
+          params:
+            path: "./0chain/docker.local/authorizer1"
+            recreate_script: "docker.local/bin/init_authorizer.sh"
+            recreate_script_workdir: "authorizer"
```
Comment (devyetii) [inline docker.local/config/conductor.zero_state_authorizer_recovery.yaml]: authorizer's data directory is not in 0chain repo, it's in token_bridge_authserver repo, so the "path" param should be: `./token_bridge_authserver/docker.local/authorizer1`, also the recreate_script should be `docker.local/bin/authorizer.init.setup.sh` and the recreate_script_workdir is the authorizer repo: `token_bridge_authserver`

File: docker.local/config/conductor.zero_state_blobber_recovery.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          timeout: "25m"
+      - command:
+          name: "fill_blobber_validator_wallets"      
+      - stop: ['blobber-1'] # Simulates Blobber failure
+      - command:
+          name: "cleanup_provider"
+          params:
+            path: "./0chain/docker.local/blobber1"
+            recreate_script: docker.local/bin/init_blobber.sh
+            recreate_script_workdir: "blobber"
```
Comment (devyetii) [inline docker.local/config/conductor.zero_state_blobber_recovery.yaml]: Same here, please figure out the correct values from the blobber repo.

File: docker.local/config/conductor.zero_state_validator_recovery.yaml [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+          timeout: "25m"
+      - command:
+          name: "fill_blobber_validator_wallets"
+      - stop: ['validator-1']
+      - command:
+          name: "cleanup_provider"
+          params:
+            path: "./0chain/docker.local/validator1"
+            recreate_script: "docker.local/bin/init_validator.sh"
+            recreate_script_workdir: "validator"
```
Comment (devyetii) [inline docker.local/config/conductor.zero_state_validator_recovery.yaml]: Same as above.

**Review by devyetii — COMMENTED** [review]
Please check them out.

**Review by devyetii — APPROVED** [review]
LGTM

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**